Cargo.toml

```
@@ -3,6 +3,10 @@ members = ["ampd", "contracts/*", "packages/*"]
 
 [workspace.dependencies]
 connection-router = { version = "^0.1.0", path = "contracts/connection-router" }
+cosmwasm-std = "1.4.0"
```

```suggestion
cosmwasm-std = "1.3.0"
```
wasmd version v0.33 used in core supports upto 1.3